packages/ui/src/components/page/details/Chart.tsx

```diff
@@ -158,7 +158,7 @@ const Chart = ({ datasets, multipleLines, selectedMetric, setSelectedMetric }: C
                       as="button"
                       key={metric}
                       onClick={() => setSelectedMetric(metric)}
-                      className="min-w-[150px] rounded-[5px] p-2 text-left text-14 text-gray-100 hover:bg-gray-600"
+                      className="text-14 min-w-[150px] rounded-[5px] p-2 text-left text-gray-100 hover:bg-gray-600"
```

Comment:
All these format changes here and below look to me as if either the tailwind config is not used properly or the prettier tailwind plugin is not working correctly, maybe check that again, it was set up correctly before and the tailwind config didn't change so there should be no format changes related to tailwind in the ui package

Reply:
I just executed eslint fix, it did that